x-pack/plugins/observability_solution/apm/scripts/test/api.js

```diff
@@ -102,11 +102,11 @@ if (server) {
 const cmd = [
   'node',
   ...(inspect ? ['--inspect-brk'] : []),
-  `../../../../../scripts/${ftrScript}`,
+  `../../../../../../scripts/${ftrScript}`,
```

Comment:
@CoenWarmer just a FYI. I fixed the static path on APM, but I think we must look at other solutions. Some scripts my be broken due to the folder migration.